DatadogCore/Sources/Core/Context/CarrierInfoPublisher.swift

```diff
@@ -11,11 +11,7 @@ import DatadogInternal
 
 import CoreTelephony
 
-// MARK: - iOS 12+
-
-/// Carrier info provider for iOS 12 and above.
 /// It reads `CarrierInfo?` from `CTTelephonyNetworkInfo` only when `CTCarrier` has changed (e.g. when the SIM card was swapped).
-@available(iOS 12, *)
 internal struct iOS12CarrierInfoPublisher: ContextValuePublisher {
```

Comment:
nit: might it make sense in the future to remove the mention of iOS 12 in the name?

Reply:
Fair call 👍, done ✅